### ISSUE TITLE
Option to modify text direction in headers

### ISSUE THREAD

**Issue by yesoreyeram** [yesoreyeram/yesoreyeram-boomtable-panel#50]

**Comment by Frakenz** [comment]
We have recently started using Boom Table and would love to see this option implemented. Right now we have a table which has multiple columns and their names are not short. It would be perfect for us if there was a checkbox that we could mark so that the header titles rotate 90 degrees clockwise.
[This is how it looks right now](https://imgur.com/a/nxMe0mK), and we are hoping to display even more columns.

**Comment by yesoreyeram** [comment]
Yes; This is in my waiting list for a long time. I am hoping to release this feature as part of next release. ( hopefully mid of feb )

Uploading the image here for the reference. 

![image](https://user-images.githubusercontent.com/153843/51756721-986f9980-20b9-11e9-925f-77275fb450a6.png)
